### PR TITLE
iOS 1.7.1 release

### PR DESCRIPTION
**Pull request by pavel-kroh-smartlook**

### CODE REVIEW

**Review by SenNeonoveNoci — APPROVED** [review]

File: src/sdk-installation/ios.mdx [hunk 1]
```diff
@@ -10,7 +10,27 @@ import { usePlatforms } from 'hooks'
 
 <Title>iOS SDK Installation</Title>
 
-1. In projects that use **Cocoapods**, Smartlook can be installed by adding the respective pod into `Podfile`:
+## Installation
+
+There are three ways how to integrate Smartlook into your app:
```
Comment (adamblack) [inline src/sdk-installation/ios.mdx:15]: ... three ways to ...

File: src/cookbook/swift-package-app-store-mach-o-error.mdx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+<TextBlock kind="important">{`
+This guide may apply to all frameworks that create or generate iOS applications and use Swift Package Manager.
+`}</TextBlock>
+
+When uploading your app with embedded Smartlook or other static library Swift Package, you may experience the
+
+`Found an unexpected Mach-O header code: 0x72613c2`
+
+error that protects you from uploading the app to App Store.
```
Comment (adamblack) [inline src/cookbook/swift-package-app-store-mach-o-error.mdx:21]: ... to the App Store.

**Review by adamblack — CHANGES_REQUESTED** [review]

File: src/sdk-installation/ios.mdx (hunk 1 above)
Comment (pavel-kroh-smartlook) [reply src/sdk-installation/ios.mdx]: fixed

File: src/cookbook/swift-package-app-store-mach-o-error.mdx (hunk 2 above)
Comment (pavel-kroh-smartlook) [reply src/cookbook/swift-package-app-store-mach-o-error.mdx]: fixed

**Review by adamblack — APPROVED** [review]

**Pull request merged**